build.gradle

```diff
@@ -83,6 +83,11 @@ allprojects {
     }
 }
 
+configurations {
+    dokkapath
+    dokkapath.description = "dokka project classpath"
+}
+
```

Comment:
I assume we can remove `dokkapath` now. Nothing is added to it and we can just pass `[]` to `processConfigurations` in the dokka task?

Reply:
👍 I removed this and left out `processConfigurations. Seems fine :).